testing/images/tekton-pod/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:latest
```

Comment:
Can we maintain these binaries from KIT?

Reply:
I can imagine our requirements will drift from KITs requirements as time goes on. I want to include the Dockerfiles used in the tekton pod examples so when replicating this, a user doesn't have to create a Dockerfile in addition to building and pushing it.